config.ini

```diff
@@ -0,0 +1,44 @@
+
```

Comment:
The convention is to name this file with `cfg` extension. Please see the examples in [python docs](https://docs.python.org/3/library/configparser.html#configparser.ConfigParser)

Reply:
Sir,I have already made that change in the present commit